themes/inbeat/layouts/_default/baseof.html

```diff
@@ -9,6 +9,7 @@
 		{{ partial "site-header.html" . }}
 		{{ $content :=  `<div class="content-video"><iframe src="https://player.vimeo.com/video/708669828?h=057cffcccb&color=ffe9f0&title=0&byline=0&portrait=0"  style="width:100%;height:100%;" frameborder="0" allow="autoplay; fullscreen; picture-in-picture" allowfullscreen></iframe></div><div class="btn-cta"><a href="https://app.inbeat.co/get-started" target="_blank"><button class="btn"><span>Try it now</span> — it's free</button></a></div>` }}
 		{{ partial "site/popup.html" (dict "context" . "id" "videoPopup" "content" $content "canClose" true  ) }}  
+    {{ partial "site/showcase-popup.html" }}
```

Comment:
Please indent with the rest if possible

Reply:
Hmm not sure. It's correctly indented in my IDE - I'll try to cleanup a bit see if it fix it